web/quiz_views.py

```
@@ -16,8 +16,167 @@
+        "web/quiz/quiz_form.html",
+        {
+            "form": form,
+            "title": "Create Session Quiz" if session else "Create Course Exam",
+            "course": course,
+            "session": session,
+            "exam_type": exam_type,
+        },
+    )
+
+@login_required
+def add_question_specialized(request, quiz_id):
```

Ruff (0.8.2)

`add_question_specialized` is too complex (14 > 10)

(C901)